### ISSUE TITLE
FBX - Actor tab in FBX Settings does not have Mesh selection field like in Documentation.

### ISSUE THREAD

**Issue by GaianHelmers** [o3de/o3de#17522]
**Describe the bug**
The Actor Groups in FBX Settings does not have the "Meshes" section of its actor definition. Preventing the ability to define subsets of the FBX as different actors.

**Assets required**
An FBX.

**Steps to reproduce**
1. Import fbx to project.
2. Select and go to FBX Settings tab.
3. Go to actor tab.
4. Try to define a specific subset of meshes for the actor.
5. There is no option.

**Expected behavior**
A mesh selection option to be able to determine which meshes apply as said actor.

**Actual behavior**
I cannot define multiple actors out of one FBX file. Need to export one for every actor piece.

**Screenshots/Video**
Screenshot of FBX Settings with no Mesh Selection.
![image](https://github.com/o3de/o3de/assets/55327922/38ca479e-f6bb-428d-bc36-2ddf0a33f03c)

Discord Discussion: https://discord.com/channels/805939474655346758/855167846240485396/1208073612729254009

**Found in Branch**
6.2.1

**Comment by lumberyard-employee-dm** [comment]
It looks like the "Select Meshes" field documentation was ported from the Lumberyard documentation: https://docs.o3de.org/docs/user-guide/visualization/animation/actor-multiple-skin/#using-a-single-fbx-file

**Comment by GaianHelmers** [comment]
What is the point of making multiple actors out of one fbx if you cant assign a mesh to the actor?

**Comment by jhanca-robotecai** [comment]
Adding `sig/content` - this part should be restudied by the animation team.

**Comment by adamdbrw** [comment]
To our understanding at the triage, this issue is to be resolved by updating the screen to show correct options.

**Comment by GaianHelmers** [comment]
I think this is only a partial issue. The meshes are actually selectable at the bottom, below the entire bone structure.
It was just misleading by having the countless bones ahead of the meshes.

![image](https://github.com/o3de/o3de/assets/55327922/e265f812-b911-4086-ad2e-2521953a6f3c)

**Comment by nick-l-o3de** [comment]
I was digging in this code and I think I see whats going on here.  As far as I can tell, the actor asset only contains the bones and nothing else, so it contains only the nodes under that skeleton.

However, the mesh group in the meshes tab (as opposed to the actor tab) is responsible for actually extracting the actual mesh data, including skinning data.   And there, you can select which meshes to include in the data.

It splits the output files into seperate streams, too, meaning, it will result in a stream of triangles, and a seperate stream of uvs, tangents, normals, and then a seperate stream of skinning data if available, too.  At runtime it binds only the ones used - so if you bring that mesh in as an unanimated prop, it will only use some of those streams, and if you bring it in as an actor, it will use the skinning data stream if available. 

As far as I can tell, it also uses a remapping table so that only the actual animated bones / bones with mesh data are involved in computations.

Basically this means that the mesh selection gui is not relevant to the actor asset panel, and you can control which meshes end up being involved in the mesh group panel instead.
